.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,10 +1,8 @@
 ---
 name: Question
-about: Do you have a generic question about Backdrop CMS?
+about: Do you have a generic question about Backdrop CMS, have you tried the BackdropCMS public forum (https://forum.backdropcms.org/)?
```

Comment:
Hey @stpaultim, thanks for the PR 👍 

The section that you have updated actually is the "description" of the issue type. As in what shows up here:

<img width="817" alt="screen shot 2018-12-14 at 12 27 00 am" src="https://user-images.githubusercontent.com/2423362/49971441-2b53da80-ff37-11e8-97fc-7777baed7890.png">

...and I am not sure that links will be rendered as links there any way.

To add something to the actual text that is shown when the user is creating an issue of this type, you need to add what it is you meant to add bellow the `---`. See this for an example:

https://github.com/backdrop/backdrop-issues/edit/master/.github/ISSUE_TEMPLATE/feature_request.md